packages/admin-ui/src/text-input/text-input.tsx

```diff
@@ -26,16 +26,24 @@ export const TextInput = forwardRef(
     } = props
 
     const id = useId(defaultId)
+    const innerRef = useRef<HTMLInputElement>(null)
+
+    const focus = () => {
```

Comment:
Focus on the input if the container is clicked. This will avoid confusion while clicking on the prefix and suffix.